react/Header/Navigation/Navigation.less

```diff
@@ -54,3 +56,16 @@
     box-shadow: inset 0 -2px @sk-white;
   }
 }
+
+.link_smallText {
+  font-size: 16px;
+}
+
+.newBadge {
+  margin-left: 5px;
+  @media only screen and (max-width: (@grid-container-width + @grid-gutter-width)) {
```

Comment:
For simplicity i dont see the need to add a single gutter width here. The container width seems to work fine IMHO.